webdev3.html

```diff
@@ -0,0 +1,115 @@
+        html, body, #map {
+            height: 100%;
+            width: 100%;
+        }</style>
+</head>
+
+<body>
+        <p>GIS 4533: Samuel Sharp</p>
+        <a href="https://leafletjs.com/">I built this with Leaflet</a>
+<div id="map"></div>
+<!-- Add the Leaflet JavaScript library -->
+<script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>    
+
+    <script> 
+    
+    //Create variable to hold map element, give initial settings to map
+    
+    //Add new basemap
+        
+        var map = L.map('map',{ center: [46.77035, -92.12242], zoom: 9});
+    L.tileLayer('https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png', {attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'}).addTo(map);
+        
+    </script>
+        
+        
+    <script>
+        //Create point feature for Split Rock Lighthouse
+        var marker = L.marker([47.2000, -91.3670]).addTo(map);
+        
+        marker.bindPopup("This is Split Rock Lighthouse").openPopup();
+        
+    </script>
+    
+    <script>
+        
+         //Create point feature for Enger Tower
+        var marker = L.marker([46.7761, -92.1250]).addTo(map);
+          marker.bindPopup("This is Enger Tower").openPopup();
+        
+        </script>
+    
+    <script>
+        
+        //Create point feature for Hawk Ridge
+        var marker = L.marker([46.8460, -92.0333]).addTo(map);
+          marker.bindPopup("This is Hawk Ridge").openPopup();
+        
+    </script>
+    
+    <script> 
+            //Create point feature for Duluth
+            
+            var marker = L.marker([46.77035, -92.12242]).
+        addTo(map);
+            
+            marker.bindPopup("This is Duluth").openPopup(); 
+        </script>
+        
+    <script>
+    //Mouse move
+L.control.mousePosition(position).addTo(map)    
+    
+    
+</script>    
+      
+    
+    
+    <script>
+    
+    //Create an Empty Popup
+        var popup = L.popup();
+        
+    //Write function to set Properties of the Popup
+        function onMapClick(e) {
+            popup
+                .setLatLng (e.latlng)
+                .setContent("You clicked the map at " + e.latlng.toString())
+                .openOn(map);
+            
+        }
+        
+        //Listen for a click event on the Map element
+        map.on('click', onMapClick);
+        </script>
+    
+     <!-- csv data and airport icons -->
+     <script 
+        
+    var airportIcon = L.Icon({
+        iconUrl: 'airport.png', iconSize: [50,40]});
+             
+    var customLayer = L.geoJson(null ,{pointToLayer: function(feature, latlng) {return L.marker(latlng, {icon: airportIcon});}});
+             
+    var airportsLayer = omnivore.csv('https://raw.githubusercontent.com/gis4533-fall2019/webdev-unit/master/airports.csv', null, customLayer).addTo(map);
+        
+         ></script>
```

Comment:
This code won't work because it's all within the tag portion (`<>`) of a `<script>`.
Move the `>` from line 112 and put it at the end of `<script` on line 103